--- venv/lib/python3.9/site-packages/glyphsLib/builder/axes.py	2021-03-14 03:32:17.000000000 +0800
+++ axes.py	2021-03-14 03:34:47.000000000 +0800
@@ -77,13 +77,19 @@
         try:
             value = _nospace_lookup(WEIGHT_CODES, user_loc)
         except KeyError:
-            return None
+            try:
+                value = int(user_loc)
+            except ValueError:
+                return None
         return class_to_value("wght", value)
     elif axis_tag == "wdth":
         try:
             value = _nospace_lookup(WIDTH_CODES, user_loc)
         except KeyError:
-            return None
+            try:
+                value = int(user_loc)
+            except ValueError:
+                return None
         return class_to_value("wdth", value)
 
     # Currently this function should only be called with a width or weight
